### PR TITLE
GPU installation for mac and ubuntu

### PR DESCRIPTION
**Pull request by kevinthesun**

### CODE REVIEW

File: docs/get_started/ubuntu_setup.md [hunk 1]
```diff
@@ -3,30 +3,63 @@ MXNet currently supports Python, R, Julia, and Scala. For users of Python and R
...
+    # Clone mxnet repository. In terminal, run the commands WITHOUT "sudo"
+    git clone https://github.com/dmlc/mxnet.git ~/mxnet --recursive
+  
+    # If building with GPU, add configurations to config.mk file:
+    cd ~/mxnet
+    cp make/config.mk .
+    echo "USE_CUDA=1" >>config.mk
+    echo "USE_CUDA_PATH=/usr/local/cuda" >>config.mk
+    echo "USE_CUDNN=1" >>config.mk
+    echo "USE_BLAS=openblas" >>config.mk
```
Comment (piiswrong) [inline docs/get_started/ubuntu_setup.md]: the default is atlas

File: docs/get_started/osx_setup.md [hunk 2]
```diff
@@ -39,27 +39,68 @@ Install the dependencies, required for MXNet, with the following commands:
 ```
 After you have installed the dependencies, use one of the following options to pull the MXNet source code from Git and build MXNet. Both options produce a library called ```libmxnet.so```.
 
+### Prepare Environment for GPU Installation
+
+If you plan to build with GPU, you need to set up environemtn for CUDA and CUDNN.
```
Comment (howard0su) [inline docs/get_started/osx_setup.md]: CUDNN is called cuDNN by nVidia.

File: docs/get_started/osx_setup.md [hunk 3]
```diff
@@ -39,27 +39,68 @@ Install the dependencies, required for MXNet, with the following commands:
...
+```
+
+Reload ~/.bash_profile file and install dependecies:
+```bash
+    . ~/.bash_profile
+    brew install coreutils
+    brew tap caskroom/cask
+```
+
+Then download [cudnn 5](https://developer.nvidia.com/cudnn).
```
Comment (howard0su) [inline docs/get_started/osx_setup.md]: cuDNN

File: docs/get_started/osx_setup.md [hunk 4]
```diff
@@ -39,27 +39,68 @@ Install the dependencies, required for MXNet, with the following commands:
...
+
+Unzip the file and change to cudnn root directory. Move the header and libraries to your local CUDA Toolkit folder:
```
Comment (howard0su) [inline docs/get_started/osx_setup.md]: header files

File: docs/get_started/ubuntu_setup.md [hunk 5]
```diff
@@ -3,30 +3,61 @@ MXNet currently supports Python, R, Julia, and Scala. For users of Python and R
...
+
+Then download [cudnn 5](https://developer.nvidia.com/cudnn).
+
+Unzip the file and change to cudnn root directory. Move the header and libraries to your local CUDA Toolkit folder:
+
+```bash
+    tar xvzf cudnn-8.0-linux-x64-v5.1-ga.tgz
+    sudo cp -P cuda/include/cudnn.h /usr/local/cuda/include
+    sudo cp -P cuda/lib64/libcudnn* /usr/local/cuda/lib64
+    sudo chmod a+r /usr/local/cuda/include/cudnn.h /usr/local/cuda/lib64/libcudnn*
```
Comment (howard0su) [inline docs/get_started/ubuntu_setup.md]: shall u run 'sudo ldconfig' to refresh libs after you copy new ones into it?

Comment (kevinthesun) [inline docs/get_started/ubuntu_setup.md]: have updated this in "Small fix" commit

**Pull request merged**